x-pack/platform/plugins/shared/security/public/management/api_keys/api_keys_grid/api_keys_table.tsx

```diff
@@ -595,10 +595,10 @@ export const categorizeAggregations = (aggregationResponse?: ApiKeyAggregations)
       : [];
 
     typeBuckets.forEach((type) => {
-      typeFilters.push(type.key);
+      typeFilters.push(type.key as CategorizedApiKey['type']);
     });
     usernameBuckets.forEach((username) => {
-      usernameFilters.push(username.key);
+      usernameFilters.push(`${username.key}`);
```

Comment:
>[!NOTE]
> Just for the record, in case someone in the future tries to understand why we do this - at the time of this PR, `key` is defined as `type FieldValue = long | double | string | boolean | null`, but `usernameFilters` expects only strings.

Reply:
To be clear, the reason it worked until now is that it used to be `type FieldValue = long | double | string | boolean | null | any`, which bypassed all type validation. The new type is more correct, but unfortunately caused those failures.